woocommerce/pinterest-for-woocommerce
# Add key flows documentation

### Changes proposed in this Pull Request:

Closes #132.
Closes #173.

Add testing flows documentation

### Screenshots:

### Detailed test instructions:
1. Go to the plugin's repo https://github.com/woocommerce/pinterest-for-woocommerce.
2. Checkout this branch.
3. Check the documentation on '/docs/testing-flows.md'

### Additional details:

### Changelog entry
> Add - Key flows documentation.
